cl/lib/http.py

```diff
@@ -11,4 +11,6 @@ def is_ajax(request: HttpRequest) -> bool:
     :param request: The HttpRequest from the user.
     :return: True if ajax, else False
     """
-    return request.META.get("HTTP_X_REQUESTED_WITH") == "XMLHttpRequest"
+    if request.META.get("HTTP_X_REQUESTED_WITH") == "XMLHttpRequest":
+        return True
+    return request.META.get("HTTP_HTTP_X_REQUESTED_WITH") == "XMLHttpRequest"
```

Comment:
What's this about?

Reply:
Might be a bug in the test client or something adding an extra HTTP prefix?

Reply:
made a change that might get rid of the need for this workaround

Reply:
Yeah, so looks like this is due to [asgi autoprefixing headers](https://github.com/django/django/blob/5.0/django/core/handlers/asgi.py#L101) which doesn't appear to be something the wsgi handler does, so just calling the test methods without the `HTTP_` prefix should get us the correct result in `request.META`.